### ISSUE TITLE
Signature could not be found when trying to whitelist.

### ISSUE THREAD

**Issue by maayant** [extremeshok/clamav-unofficial-sigs#42]
### Virus found notification from Amavis:
A virus was found: **MBL_575906.UNOFFICIAL**
Scanner detecting a virus: ClamAV-clamd

### Trying to whitelist and getting the error:
```
[root@mailgw1 ~]# /usr/local/bin/clamav-unofficial-sigs.sh -w
################################################################################
 eXtremeSHOK.com ClamAV Unofficial Signature Updater
 Version: v4.9.2 (02 December 2015)
 Copyright (c) Adrian Jon Kriel :: admin@extremeshok.com
################################################################################
================================================
Loading config: /etc/clamav-unofficial-sigs.conf
================================================

Input a third-party signature name that you wish to whitelist due to false-positives
and press enter (do not include '.UNOFFICIAL' in the signature name nor add quote
marks to the input string):
MBL_575906
Signature 'MBL_575906' could not be found.
This script will only create a whitelise entry in my-whitelist.ign2 for ClamAV
'UNOFFICIAL' third-Party signatures as found in the *.ndb databases.
```

### Verifying that the virus name and signature exist in the database:
```
[root@mailgw1 ~]# grep MBL_575906 /var/lib/clamav/*
malwarepatrol.db:MBL_575906=616d6170726f2e616d61646575732e636f2e696c2f616d61736f66742f73637269707473
```

Manually creating the my-whitelist.ign2 file and adding "MBL_575906" inside it works as expected, Clam ignores the signature.

**Comment by extremeshok** [comment]
Will code a fix asap

**Comment by extremeshok** [comment]
Fixed with b718125f1146c6f9ee91b65ba6ed7c7437326f51
